reference.md

```diff
@@ -704,3 +707,73 @@ Field Name | Presence | Type | Description
   }
 }
 ```
+
+### booking_rules.json
+
+This file defines rules about how to book a ride in advance and is to be used instead `wait_times` if users can not order a ride in real-time. With ride bookings, no wait times are applied. If available, users can either book a ride for the same day with an advance notice or for a futur day. 
```

Comment:
Agreed for the first point, those fields are helpful. 

Ambivalent on the second point. 
* At the minimum, `wait_times.geojson` provides static wait time. It'd be a weird situation to be able to book for a specific time, but assuming at the same time that the ride will always have 15 minutes delay. 
* For `wait_times`, that provides real-time wait times, shouldn't your wait time for a pre-book ride be specific to your booking, and not a "generic" wait times? There's a difference between a wait time between two locations, and a wait time between two locations for which you have a reservation already. I am not sure how to express this however. 

> but at the same time may face delays to pick up the riders on time (eg: facing a 2 hours' delay, because of occasional high ridership demand).
I also assume in general that agencies pre-booking feature will try to first confirm that they don't overbook their vehicles either. So a surge in demand would simply results in unsuccessful pre-booking for the user.

Reply:
What you say is true in theory @timMillet but I'm not sure that's the case in practice. Either you have a wait time with real time booking or you book for later. I've never seen both concept mixed.

Reply:
The PR as it makes sense if GOFS-lite is a stand-alone; in a real-time feed, wait time and booking time can overlap. The issue that I see is mostly related to compatibility with GTFS-Flex and GTFS-OnDemand, more specifically booking_rules.txt and wait_rules.txt.
In these static files, there is nothing that forbid an on-demand service operator to provide a general booking rule valid at all times, and wait times that differ depending on dates, days, time, and locations. How would we translate that using the spec from this PR?